dd-sdk-android/src/main/java/com/datadog/opentracing/DDTracer.java

```diff
@@ -643,7 +649,7 @@ private DDSpanContext buildSpanContext() {
                     tags.putAll(((TagContext) parentContext).getTags());
                     origin = ((TagContext) parentContext).getOrigin();
                 } else {
-                    origin = null;
+                    origin = this.origin;
```

Comment:
Just double checking - this `origin` is used for `x-datadog-origin` header when injecting span context into `Request`, right? I don't see this header setup in the scope of this PR, I guess it's in-build into `dd-trace-java`?

Reply:
Yes it's built in the `dd-trace-java` code.